wsgidav/debug_filter.py

```diff
@@ -142,8 +142,7 @@ def __call__(self, environ, start_response):
             # _logger.info("<{}> --- {} Request ---".format(
             #         threading.currentThread().ident, method))
             for k, v in environ.items():
-                if k == k.upper():
-                    _logger.info("{:<20}: '{}'".format(k, safe_re_encode(v, "utf8")))
+                _logger.info("{:<20}: '{}'".format(k, safe_re_encode(str(v), "utf8")))
```

Comment:
I think `str(v)` may raise an encoding exception, so we should handle it with try/except. See the proposed implementation in the PR comments instead